packages/api-client/package.json

```diff
@@ -21,6 +21,7 @@
     "directory": "packages/api-client"
   },
   "dependencies": {
+    "@tupaia/server-boilerplate": "workspace:*",
```

Comment:
Is this a circular dependency now? Is that not an issue with workspace imports?

Looks like we're just after the `ExpressResponse` type :thinking: might be a little awkward but we could import the `@types/express` package into the `@tupaia/types` package and define the `ExpressRequest/Response` types there instead?

Reply:
Turns out I only need the `Response` type from `express` which I installed the `express` dependency and removed this `@tupaia/server-boilerplate`!